src/main/java/com/snowflake/kafka/connector/internal/SnowflakeConnectionServiceV1.java

```diff
@@ -351,7 +351,7 @@ public boolean isTableCompatible(final String tableName) {
       }
       compatible = hasMeta && hasContent && allNullable;
     } catch (SQLException e) {
-      LOGGER.debug("table {} doesn't exist", tableName);
+      LOGGER.info("Table {} doesn't exist. Exception {}", tableName, e.getStackTrace());
```

Comment:
That message in logs can be interpreted as a serious situation without context. I would keep it on debug. The next one is much better for INFO.